src/main/java/unicash/logic/commands/SummaryCommand.java

```diff
@@ -12,11 +12,15 @@ public class SummaryCommand extends Command {
 
     public static final String MESSAGE_USAGE = CommandType.SUMMARY.getMessageUsage();
 
-    public static final String SHOWING_SUMMARY_MESSAGE = "Opened UniCa$h summary window.";
+    public static final String MESSAGE_SHOWING_SUMMARY_SUCCESS = CommandType.SUMMARY.getMessageSuccess();
+
+    public static final String MESSAGE_SHOWING_SUMMARY_FAILURE = CommandType.SUMMARY.getMessageFailure();
```

Comment:
I think a minor issue I have with this is that the message isn't actually a failure right? Like we don't throw a `CommandException` as a result of not having any expenses. So I'm thinking this should just be a separate string constant, much like `GetBudgetCommand`

Reply:
Makes sense. In that case, then I will have to make some changes to the UG as well. I initially deemed the event that the summary window fails to pop up as a failure, but you're right in that it doesn't throw any exception, but it's just the way that it was implemented. 

Will change the code in this PR and modify this part in the UG as well!